VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/tests/waiting-room-base.component.non-event.spec.ts

```diff
@@ -100,38 +100,46 @@ describe('WaitingRoomComponent message and clock', () => {
 
         it('should switch Participants from false to true  when starting from false', () => {
             // Arrange
+            component.panelStates[chatPanelName] = false;
```

Comment:
```suggestion
            component.panelStates[chatPanelName] = true;
```

Reply:
If chat panel is true it should become false in this case right?

Reply:
Added couple more tests